node/runtime/src/lib.rs

```diff
@@ -1060,6 +1060,44 @@ impl pallet_assets::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub const RenvmBridgeUnsignedPriority: TransactionPriority = TransactionPriority::max_value() / 16;
+
+	pub const RenVMModuleId: ModuleId = ModuleId(*b"RenToken");
+}
+
+impl edge_ren::Config for Runtime {
+	type Event = Event;
+	type RenvmBridgeUnsignedPriority = RenvmBridgeUnsignedPriority;
+	type ControllerOrigin= frame_system::EnsureOneOf<AccountId,
+		pallet_collective::EnsureProportionAtLeast<_2, _3, AccountId, CouncilCollective>,
+		frame_system::EnsureRoot<AccountId>,
+	>;
+	type ModuleId= RenVMModuleId;
+	type Assets = EdgeAssets;
+	type WeightInfo = edge_ren::weights::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+	pub const AssetsAllowFreezing: bool = true;
+	pub const AssetsAllowBurning: bool = true;
+	pub const AssetsAllowMinting: bool = true;
+}
+
+impl edge_assets::Config for Runtime {
+	type Currency = Balances;
+	type Event = Event;
+	type Balance = Balance;
+	type AssetId = u32;
+	type ForceOrigin = EnsureOneOf< AccountId, EnsureRoot<AccountId>, edge_ren::EnsureRenVM<Runtime>>;
```

Comment:
I would rather this bridge not be included in the ForceOrigin because it is mocked through the Controller Origin.

Reply:
edge_ren::EnsureRenVM doesn't yet do anything. There are no raw calls that it dispatches.